sdk/rust/oak/src/io/receiver.rs

```diff
@@ -16,6 +16,10 @@
 
 use crate::{io::Decodable, ChannelReadStatus, OakError, OakStatus, ReadHandle};
 use log::error;
+use prost::{
+    bytes::{Buf, BufMut},
+    encoding::{DecodeContext, WireType},
+};
 use serde::{Deserialize, Serialize};
```

Comment:
Can we now remove the `Serialize`/`Deserialize` implementations for `Handle`, `ReadHandle`, `WriteHandle`, `Sender` and `Receiver` and fix #495 along the way?  (Follow-up PR, obvs.)